src/main/kotlin/io/sip3/captain/ce/pipeline/RouterHandler.kt

```diff
@@ -50,18 +50,26 @@ class RouterHandler(vertx: Vertx, bulkOperationsEnabled: Boolean) : Handler(vert
             // RTP or RTCP packet
             val packetType = buffer.getUnsignedByte(offset + 1).toInt()
             if (packetType in 200..211) {
-                rtcpHandler.handle(buffer, packet)
+                // Doesn't matter if RTCP packet was rejected or not
```

Comment:
I think that if comment is correct, we don't need to check `packet.rejected` below in code.

Reply:
Modified comment message